pkg/ebpf/tracee.go

```diff
@@ -1070,3 +1081,19 @@ func (t *Tracee) invokeInitNamespacesEvent() {
 		t.stats.EventCount.Increment()
 	}
 }
+func (f *Filter) getTracedIfaceIdx(ifaceName string) int {
```

Comment:
Unlike `getCapturedIfaceIdx` which has a meaning for the index (which is the index of the NgInterface), I don't think there is a meaning for the traced interface index. Instead, you can just have a function that checks if `ifaceName` is valid  (exists in InterfaceToTrace slirce) or not, and return a bool